#This message allows to comunicate camera properties
uint8 property_id
bool blue
float32 value_a
bool reset
bool targets
uint16 val_a


